themes/default/layouts/registry/api.html

```diff
@@ -4,7 +4,7 @@
     <div class="flex-col min-w-0">
         {{ partial "registry/package/header.html" . }}
         <div class="lg:flex">
-            <div class="lg:w-9/12 xxl:w-12/12">
+            <div class="xxl:w-8/12">
```

Comment:
updated this to just be `w-8/12` above xxl breakpoint because this is where the right nav gets shown, so we need to account  for that space on the screen when it gets rendered above that point. Below, that point the main content area should just expand naturally.